docs/design/fabric-v2+/fpc-management.md

```diff
@@ -0,0 +1,80 @@
+# FPC Management
```

Comment:
Do we also need something like a enclave_status command? to see if there is already an enclave up and running? or some command to helps to check if we need to call create or not?

Reply:
maybe, but I don't know what we gain at this point.
perhaps we should think about meaningful error codes for these situations.

Reply:
I guess from a fabric symmetry perspective, a `querycreatedenclave` could make sense. Implementation should be probably quite straightforward but it will (as all thing do) require a non-trivial amount of time. As it doesn't seem essential nd we always can added it on demand, i would skip it for now.

Reply:
> perhaps we should think about meaningful error codes for these situations.

I don't recall the peer cli have any well-defined (well-documented) exit codes other than being non-zero on errors.  So maybe just having somewhat meaningful error-messages should do.  (Not that exit codes wouldn't be a good idea, but i guess for our scripting we would do this immediatly after commit, so double registration shouldn't be any issue and we probably would have to bail anyway, no matter what ...?)